longevity_sla_test.py

```diff
@@ -43,7 +48,7 @@ def test_custom_time(self):
         # Wait for all SLs are propagated to all nodes
         for role in self.roles + [self.fullscan_role]:
             # self.fullscan_role may be None if "run_fullscan" is not defined
-            if role:
+            if role and is_enterprise:
```

Comment:
Why we check it in the enterprise only? Service level is allowed to be created in OSS also. Or am I wrong?

Reply:
I though that SLA(WLP) is only Enterprise feature.

Reply:
https://opensource.docs.scylladb.com/stable/using-scylla/workload-prioritization.html

Reply:
Service level can be created in OSS, just `shares` parameter is not supported. 
But because of we check shares in "wait for propagation" function, this validation should not be used in OSS.